clients/fides-js/src/fides-tcf.ts

```diff
@@ -177,18 +239,18 @@ const init = async (config: FidesConfig) => {
   // eslint-disable-next-line no-param-reassign
   config.options = { ...config.options, ...overrideOptions };
   const cookie = getInitialCookie(config);
+  let hasFidesStringOverride = !!config.options.fidesString;
   if (config.options.fidesString) {
-    // If a fidesString is explicitly passed in, we override the associated cookie props, which are then used to
-    // override associated props in experience
-    debugLog(
-      config.options.debug,
-      "Explicit fidesString detected. Proceeding to override all TCF preferences with given fidesString"
-    );
-    cookie.fides_string = config.options.fidesString;
-    cookie.tcf_consent = transformFidesStringToCookieKeys(
+    const { cookie: updatedCookie, success } = updateFidesCookieFromString(
+      cookie,
       config.options.fidesString,
       config.options.debug
     );
+    if (success) {
+      Object.assign(cookie, updatedCookie);
+    } else {
+      hasFidesStringOverride = false;
```

Comment:
this functionally works for now, but perhaps a better name would be `hasValidFidesStringOverride`?